aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsTranslateTest.java

```diff
@@ -87,7 +87,7 @@ public void testTranslationWithDefaultLanguage() throws Exception {
 
         // Assert translation
         String translation = result.getTranslatedText();
-        assertEquals("¡Hola mundo!", translation);
+        assertTrue(translation.cotains("Hola mundo"));
```

Comment:
contains, not cotains

Reply:
Ugh. One line change and I still fucked it up, haha. Fixing...